agent/engine/docker_task_engine.go

```diff
@@ -46,6 +47,8 @@ const (
 	capabilityTaskIAMRole        = "task-iam-role"
 	capabilityTaskIAMRoleNetHost = "task-iam-role-network-host"
 	labelPrefix                  = "com.amazonaws.ecs."
+	ECSDataDir                   = "/data/"
+	HostDataDir                  = "/var/lib/ecs/"
```

Comment:
These probably should be moved to `config` package, and also you may replace the one [here](https://github.com/aws/amazon-ecs-agent/blob/master/agent/config/config_unix.go#L30)